compiler/src/dmd/mars.d

```diff
@@ -592,6 +592,9 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             printf("arguments[%d] = '%s'\n", i, arguments[i]);
         }
     }
+
+    files.reserve(arguments.length - 1);
```

Comment:
This looks like an unnecessary and unrelated addition

Reply:
It comes from line 671 in main.d above.